restapi/stories.go

```diff
@@ -190,9 +190,26 @@ func (api api) CreateStoriesFormData(w http.ResponseWriter, r *http.Request) ren
 	if err := api.database.Create(&story).Error; err != nil {
 		return rest.ErrInternal(api.logger, err)
 	}
+	fmt.Println(r.FormValue("bio"))
+	names := r.Form["tags"]
+	err = api.AddTags(names, story)
+	if err != nil {
+		return rest.ErrInvalidRequest(api.logger, "Error Adding Tags", err)
+	}
 	return rest.MsgStatusOK("Story Added Successfully")
 }
 
+func (api api) AddTags(names []string, story models.Story) error {
+	var tags []models.Tag
```

Comment:
Again we can predeclare and just array index to add tags, no need to `append` if not necessary.